app/views/announcement/_new.html.erb

```diff
@@ -1,7 +1,14 @@
 <%= form_for @announcement, url: {action: "create"} do |f| %>
-  <%= f.label 'Receipent enter name goes here' %>
+  <%= label_tag(:fname, "Enter a Name of a User or select a roll(s)")%>
```

Comment:
What Rosie said for a comprehensive review..

But I also think this way of choosing a user to send to is not very user friendly. It would be really hard to remember everyone's name to send to. Honestly, for our demo tomorrow you could remove the user sending and just leave sending emails to teams. I think the individual user sending would take too much backend/front end work to create quickly and work well.